docs/source/03-07-product-guide-trust.md

```diff
@@ -3,47 +3,53 @@ title: Trust
 permalink: /trust/
 ---
 
-Trust is a set of components within Talk that incorporate automated moderation
-features based on a user's previous behavior.
+Trust is a set of components within Talk that incorporate basic automated moderation features based on a user's previous behavior. 
 
 ## User Karma Score
 
-Using Trust’s calculations, Talk will automatically pre-moderate comments of
-users who have a negative karma score. All users start out with a `0` neutral
-karma score. If they have a comment approved by a moderator, their score
-increases by `1`; if they have a comment rejected by a moderator, it decreases
-by `1`. When a commenter is labeled as Unreliable, their comments must be
-moderated before they are posted.
+Using Trust’s calculations, Talk will automatically hold back, move to the Reported queue, and tag with a 'History' marker, any comments by users who have an Unreliable karma score. (This is for sites who practice post-moderation. If you set pre-moderation of all comments sitewide, this feature has limited use.)
 
-When a commenter has one comment rejected, their next comment must be moderated
-once in order to post freely again. If they instead get rejected again, then
-they must have two of their comments approved in order to get added back to the
-queue.
+All users start out with a Neutral karma score (`0`). If they have a comment approved by a moderator, their score increases by `1`; if they have a comment rejected by a moderator, it decreases by `1`. When a commenter's score is labeled as Unreliable, their comments must be approved from the Reported queue before they are posted. Commenters are shown a message stating that a moderator will review their comment shortly.
 
 Here are the default thresholds:
 
 ```text
--2 and lower: Unreliable
--1 to +2: Neutral
-+3 and higher: Reliable
+-1 and lower: Unreliable
+0 to +1: Neutral
++1 and higher: Reliable (we don't do anything with this label right now)
```

Comment:
This should actually be +2:

https://github.com/coralproject/talk/blob/4413e5616c8655b30f08144e78606033dd4a62a5/config.js#L245

Reply:
oh, good point.